packages/teleterm/src/ui/DocumentCluster/DocumentCluster.tsx

```diff
@@ -91,12 +90,6 @@ export function Cluster() {
             {`${clusterCtx.state.clusterName}`}
           </Text>
         </Text>
-        <Flex alignItems="center">
-          <ClusterSearch onChange={clusterCtx.changeSearchValue} />
-          <ButtonPrimary ml={2} size="small" onClick={clusterCtx.sync}>
-            Sync
```

Comment:
Should we replace this button with some kind of "Refresh" button? Right now there's no way to simply repeat the request with the current params. It could use the same params sans the page – idk if it's feasible to do refresh while keeping page position given how pagination works on our clusters.

Reply:
A refresh button outside of the table would prove to be difficult as the state would need to be propped up outside of its current context. If you press 'Enter' in the search bar, it will resend the same query and 'refresh' the page with the most recent info which I think is probably the best/easiest solution right now. Also, going back into the bar and pressing 'Enter' to me is even more intuitive than a refresh button. Thoughts?

Reply:
> A refresh button outside of the table would prove to be difficult as the state would need to be propped up outside of its current context.

I think it should be doable without moving the state, perhaps through `useImperativeHandle` or a callback of some kind, no? The parent component doesn't need to use any of that state, it just so happens that the way the components are organized doesn't allow us to have the button be managed by the appropriate component itself. So IMHO it's fine to use such escape hatch in that case.

> Also, going back into the bar and pressing 'Enter' to me is even more intuitive than a refresh button.

I'd say it's far less intuitive than clicking on a button that says "Refresh" which refreshes the list. 😏

Anyway, it seems that we don't have time for that right now and it's perhaps not the most important thing in the world. The Web UI doesn't offer that functionality as well but at least the browser has a persistent reload button.

Reply:
I'm not sure if we need the refresh button. In what situations would it be useful? I think the resource lists change very rarely. For example, Slack and GitHub don't have a separate button to refresh the search results. 

Also if we decide to add a button, it seems to me that it should be placed at the table level, not above it (it would disrupt the hierarchy IMO).

Reply:
Yeah, it would be really nice to have automatic UI updates. It especially hurts us in access requests, where you don't know if your request has been accepted until you refresh the list.